website/src/pages/downloads/ProjectCard.js

```diff
@@ -44,8 +64,12 @@ const ProjectCard = (props) => {
     releaseDate,
     githubRepo,
     githubBranch,
-    downloadPath
+    downloadPath,
+    LTSDownloadPath = ' '
   } = props;
+
+  const Download = props.name === 'APISIX®' ? '2.12.1 Current' : 'Download'
```

Comment:
Should I use the data from `downloads.js`? To avoid the need to also modify the js file for each update, not every one of us releaser knows the frontend.